ScoreSaber/Core/ReplaySystem/UI/Components/AmeBar.cs

```diff
@@ -10,39 +10,26 @@
 
 namespace ScoreSaber.Core.ReplaySystem.UI.Components {
     internal class AmeBar : MonoBehaviour {
-        private static Material _mainUIFontMaterial;
-        private CurvedTextMeshPro _currentTimeText;
-
-        private CurvedTextMeshPro _endTimeText;
+        private RectTransform _rectTransform;
         private RectTransform _fillBarTransform;
 
-        private RectTransform _otherTransform;
-        private RectTransform _rectTransform;
+        private CurvedTextMeshPro _endTimeText;
+        private CurvedTextMeshPro _currentTimeText;
 
         public float barFill {
             get => _fillBarTransform.anchorMax.x;
             set => _fillBarTransform.anchorMax = new Vector2(Mathf.Lerp(-1f, 1f, value), _fillBarTransform.anchorMax.y);
         }
 
         public float currentTime {
-            set => _currentTimeText.text = $"{(int)value / 60}:{value % 60f:00}";
+            set => _currentTimeText.text = string.Format("{0}:{1:00}", (int)value / 60, value % 60f);
```

Comment:
Why are we rolling back the `string.Format`'s?

Reply:
Mistake.